SingularityUI/app/actions/api/slaves.es6

```diff
@@ -2,14 +2,14 @@ import { buildApiAction, buildJsonApiAction } from './base';
 
 export const FetchSlaves = buildApiAction(
   'FETCH_SLAVES',
-  {url: '/slaves'}
```

Comment:
turns out this is handled in staging so the `buildApiAction` doesn't return the extra slash  ![bitmoji](https://render.bitstrips.com/v2/cpanel/10211672-211453933_4-s1-v1.png?transparent=1&palette=1&width=146)